### PR TITLE
LPP-51929 Add min-height to resolve CKEditor Firefox Issue

### PR DESCRIPTION
**Pull request by fortunatomaldonado**
https://liferay.atlassian.net/browse/LPS-203446

Adding min-height resolves the Firefox specific issue that the CKEditor scrolls to top after text is modified, making it hard to see what was modified.
Please let me know if there are any questions or comments about this.

Thank you!

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: modules/apps/journal/journal-web/src/main/resources/META-INF/resources/css/ckeditor.scss [hunk 1]
```diff
@@ -1,3 +1,8 @@
 html {
 	height: 100%;
 }
+
+.cke_editable.html-editor.h-100.cke_editable_themed.cke_contents_ltr {
```
[the remark this answers is not in the document]
Comment (fortunatomaldonado) [reply modules/apps/journal/journal-web/src/main/resources/META-INF/resources/css/ckeditor.scss]: Thank you @veroglez, I was able to make the changes!

**Comment by fortunatomaldonado** [comment]
ci:test:sf

**Comment by fortunatomaldonado** [comment]
ci:test:relevant

**Comment by veroglez** [comment]
LGTM!

**Comment by veroglez** [comment]
ci:forward

**Comment by Tim-Cao** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/145186

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**